s3_bucket_block/main.tf

```diff
@@ -21,6 +21,17 @@ variable "region" {
   description = "AWS Region"
 }
 
+variable "inventory_bucket_arn" {
+  description = "ARN of the S3 bucket used for collecting the S3 Inventory reports."
+  type        = string
+}
+
+variable "sse_algorithm" {
+  description = "SSE algorithm to use to encrypt reports in S3 Inventory bucket."
+  type        = string
+  default     = "aws:kms"
```

Comment:
Is there ever a time we don't use `aws:kms` for the sse algorithm with s3? Perhaps it makes sense to share this config more globally since I see the value also used elsewhere in this repo.

Reply:
We have plenty of buckets that use `AES256`, including the inventory bucket.